src/main.py

```diff
@@ -279,36 +280,41 @@ def get_arg_parser():
         p.add_argument(
             "--base-patch-version",
             required=True,
-            help=""
+            help="Specify the base patch version from which a new version should be created."
         )
         p.add_argument(
             "--force",
             action='store_true',
-            help=""
+            help="Overwrites any existing directory corresponding to the new version that will be generated."
         )
 
     build_image_parser = subparsers.add_parser(
         "build",
-        help=""
+        help="Builds a new image from the Dockerfile."
     )
     build_image_parser.add_argument(
         "--target-patch-version",
         required=True,
-        help=""
+        help="Specify the target version of Amazon SageMaker Distribution for which an image needs to be built."
     )
     build_image_parser.add_argument(
         "--skip-tests",
         action='store_true',
-        help=""
+        help="Disable running tests against the newly generated Docker image."
     )
     build_image_parser.add_argument(
         "--target-ecr-repo",
         action='append',
-        help=""
+        help="Specify the AWS ECR repository in which this image needs to be uploaded."
+    )
+    build_image_parser.add_argument(
+        "--is-public-ecr-repo",
+        action='store_true',
+        help="Specify whether the AWS ECR repository is a public ECR repository. Default is Private ECR repository."
     )
```

Comment:
Instead of making this a user-provided input, can't the code automatically deduce this from the ECR URI? Unless I am mistaken, the public URIs always start with `public.ecr.aws/`, so wouldn't that work?

Reply:
Done. Addressed this as part of the revision